psychopy/plugins/__init__.py

```diff
@@ -764,7 +764,10 @@ def loadPlugin(plugin):
         for attr, ep in attrs.items():
             try:
                 # parse the module name from the entry point value
-                module_name, _ = ep.value.split(':', 1)
+                if ':' in ep.value:
```

Comment:
I'm not sure we even need to split by `:` given that the second part is discarded anyway, so yes this looks fine :)